components/automate-ui/src/app/modules/infra-proxy/infra-search-bar/infra-search-bar.component.ts

```diff
@@ -15,24 +13,17 @@ import {
 
 export class InfraSearchBarComponent {
   inputText = '';
+  formActive = false;
 
   @Input() placeHolder: string;
   @Output() searchButtonClick = new EventEmitter<string>();
-  @ViewChild('search_box', { static: true }) inputField: ElementRef;
 
-  handleFiltersClick(currentText: string): void {
+  onSubmit(currentText: string): void {
     this.searchButtonClick.emit(currentText);
   }
 
-  pressEnter(currentText: string): void {
-    this.searchButtonClick.emit(currentText);
+  toggleFocus(): void {
+    this.formActive = !this.formActive;
   }
 
-  handleInput(key, currentText): void {
-    switch (key.toLowerCase()) {
-      case 'enter':
-        this.pressEnter(currentText);
-        break;
-    }
-  }
```

Comment:
Author note: Unnecessary function when using form element instead for your search bar.